openconnect: add page (#4780)
tldr-pages/tldr: pages/common/openconnect.md

# openconnect

> A VPN client, for Cisco AnyConnect VPNs and others.

> > More information: <https://www.infradead.org/openconnect/>.

Reviewer: I think the link should be to the manual of the command:
```suggestion
> More information: <https://www.infradead.org/openconnect/manual.html>.
```

> - Establish a VPN connection to the specified server:

Reviewer: ```suggestion
- Connect to a server:
```

`openconnect {{vpn.example.org}}`

> - Establish a VPN connection to the specified server and continue in the background:

Reviewer: Based on `hostapd` and `mosquitto` pages:
```suggestion
- Connect to a server, forking into the background:
```

`openconnect --background {{vpn.example.org}}`

> - Terminate a VPN session that is running in the background:

Reviewer: I'm not sure about this example, because a lot of command could have this example. I vote for remote it, but you can wait a second reviewer.
```suggestion
- Terminate the connection that is running in the background:
```

Author: There is no other documentation in openconnect on how to terminate a backgrounded session. I personally found this very frustrating, because I had to go online and search for a solution. I would have saved a few minutes if this example was in the man pages, but it was not. 

Apart from that, it is apparently important to do a `killall -SIGINT` instead of a mere `killall`, because without the `-SIGINT` the previous network configuration will not be restored. Even experienced users might not expect this.

Reviewer: Oh, I see. Please apply the suggestion and keep the example for now.

# openconnect

> A VPN client, for Cisco AnyConnect VPNs and others.
> More information: <https://www.infradead.org/openconnect/manual.html>.

- Connect to a server:

`openconnect {{vpn.example.org}}`

- Connect to a server, forking into the background:

`openconnect --background {{vpn.example.org}}`

- Terminate the connection that is running in the background:

`killall -SIGINT openconnect`

> - Connect to a server, reading further options from a config file:

Reviewer: ```suggestion
- Connect to a server, reading options from a config file:
```

`openconnect --config={{path/to/file}} {{vpn.example.org}}`

> - Connect to a server and authenticate with the specified SSL client certificate:

Reviewer: ```suggestion
- Connect to a server and authenticate with a specific SSL client certificate:
```
